python/ray/services.py

```diff
@@ -20,8 +20,11 @@
 import ray
 import ray.ray_constants as ray_constants
 
-from ray.tempfile_services import (get_ipython_notebook_path, get_temp_root,
-                                   new_redis_log_file, get_logs_dir_path)
+from ray.tempfile_services import (get_ipython_notebook_path,
+                                   get_temp_root,
+                                   new_redis_log_file,
+                                   get_logs_dir_path,
+                                   )
```

Comment:
yapf should format this to 
```
from ray.tempfile_services import (
   get_ipython_notebook_path,
   get_temp_root,
   new_redis_log_file,
   get_logs_dir_path,
)
```
Also, please sort the items alphabetically. (The linter doesn't seemingly check this.)